clip_benchmark/datasets/multilingual_mscoco-old.py

```diff
@@ -0,0 +1,132 @@
+from subprocess import call
```

Comment:
why a -old and not old ?

Reply:
Woops, this filde should not have been included. It has now been removed from the commit.